nxviz/plots.py

```diff
@@ -219,14 +224,15 @@ def draw(self):
             self.draw_group_labels()
         logging.debug("DRAW: {0}".format(self.sm))
         if self.sm:
-            self.figure.subplots_adjust(right=0.8)
-            cax = self.figure.add_axes([0.85, 0.2, 0.05, 0.6])
-            self.figure.colorbar(self.sm, cax=cax)
+            #self.figure.subplots_adjust(right=0.8)
+            #cax = self.figure.add_axes([0.85, 0.2, 0.05, 0.6])
+            #self.figure.colorbar(self.sm, cax=cax)
+            self.figure.colorbar(self.sm, ax=self.ax)
         self.ax.relim()
         self.ax.autoscale_view()
         self.ax.set_aspect("equal")
 
-    def compute_node_colors(self):
+    def compute_node_colors(self, **kwargs):
```

Comment:
The `**kwargs` pattern I have seen thus far leaves options tough to document.

What are your thoughts on the following suggested change?

```suggestion
    def compute_node_colors(self, cmap: str=None):
        """
        :param cmap: The matplotlib colormap to use.
        """
```

It would then precipitate the following change in line 259 below.

Reply:
Much better, that was just a hack.

Reply:
Ok, please accept the changes; you can do so here, or just copy/paste it into your own branch.